setup.cfg

```diff
@@ -235,8 +235,6 @@ filterwarnings =
     ignore:`product` is deprecated as of NumPy 1.25.0:DeprecationWarning
     # sunpy IO skip here to prevent pytest breaking items downstream
     ignore:.*module is deprecated, as it was designed for internal use
-    # This is raised when the VSO redirects and we do not want this to stop the CI
-    ignore::sunpy.util.exceptions.SunpyConnectionWarning
```

Comment:
Have to see if the online test needs this.

Reply:
We need this here because otherwise the VSO tests fail when they fall over to the second endpoint rather than passing.